File: emulinker/src/main/java/org/emulinker/kaillera/model/event/GameStatusChangedEvent.java
package org.emulinker.kaillera.model.event;

import org.emulinker.kaillera.model.*;

public class GameStatusChangedEvent implements ServerEvent {
  private KailleraServer server;
  private KailleraGame game;

  public GameStatusChangedEvent(KailleraServer server, KailleraGame game) {
    this.server = server;
    this.game = game;
  }

  public String toString() {
    return "GameStatusChangedEvent";
  }

  public KailleraServer getServer() {
    return server;
  }

  public KailleraGame getGame() {
    return game;
  }
}

File: emulinker/src/main/java/org/emulinker/kaillera/model/event/GameStatusChangedEvent.kt
package org.emulinker.kaillera.model.event

import org.emulinker.kaillera.model.KailleraGame
import org.emulinker.kaillera.model.KailleraServer

data class GameStatusChangedEvent(override val server: KailleraServer, val game: KailleraGame) :
    ServerEvent {
  override fun toString(): String {
    return "GameStatusChangedEvent"
  }
}
